infrastructure/images/kafka/scripts/configure-kafka.sh

```diff
@@ -29,3 +34,12 @@ done
 
 echo "Printing out the broker configuration"
 cat /etc/kafka/server.properties
+
```

Comment:
The following will need to run only if the kafka storage hasn't been initialized.